summoner-cli/src/Summoner/Project.hs

```diff
@@ -155,25 +157,61 @@ generateProject settingsNoUpload isOffline projectName Config{..} = do
         Last prelude@(Just (CustomPrelude p _)) ->
             prelude <$ successMessage ("Custom prelude " <> p <> " will be used in the project")
 
+    getNixPkgSet :: Bool -> IO (Maybe NixPkgSet)
+    getNixPkgSet usingNix = if usingNix
+        then case cNixPkgSet of
+            Last Nothing -> do
+                let yesDo, noDo :: IO (Maybe NixPkgSet)
+                    yesDo = do
+                        o <- queryNotNull "Nix package set GitHub owner name: "
+                        r <- queryNotNull "Nix package set GitHub repo name: "
```

Comment:
The same quesion here.. And probably they are linked together so we can make it in pairs?
This doesn't mean that it should be done now, just to know for the future.